src/components/perfomers.tsx

```diff
@@ -0,0 +1,62 @@
+"use client";
```

Comment:
i dont think we need to make this a client component. you can make the component itself async, and then await the api response before returning the jsx